test/plot/templates/plot_templates_AIOCG.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+import matplotlib.pyplot as plt
+from pyrolite.plot.templates.AIOCG import AIOCG
+
+
+class TestAIOCGPlot(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def test_TAS_default(self):
```

Comment:
These method names still refer to TAS; they should be updated - you can leave out the name if you prefer and just refer to what kind of test is being performed (e.g.`test_default`, `test_relim`).